src/sidebar/components/test/help-panel-test.js

```diff
@@ -1,45 +1,197 @@
+    return mount(
+      <HelpPanel auth={fakeAuth} session={fakeSessionService} {...props} />
+    );
+  }
 
-  beforeEach(function() {
+  beforeEach(() => {
+    fakeAuth = {};
+    fakeSessionService = { dismissSidebarTutorial: sinon.stub() };
     fakeStore = {
-      frames: sinon.stub().returns([]),
+      getState: sinon
+        .stub()
+        .returns({ session: { preferences: { show_sidebar_tutorial: true } } }),
+      mainFrame: sinon.stub().returns(null),
+    };
+    fakeVersionDataObject = {
+      asEncodedURLString: sinon.stub().returns('fakeURLString'),
     };
+    fakeVersionData = sinon.stub().returns(fakeVersionDataObject);
+
+    HelpPanel.$imports.$mock(mockImportedComponents());
+    HelpPanel.$imports.$mock({
+      '../store/use-store': callback => callback(fakeStore),
+      '../util/version-data': fakeVersionData,
+    });
+  });
+
+  afterEach(() => {
+    HelpPanel.$imports.$restore();
+  });
+
+  context('when viewing tutorial sub-panel', () => {
+    it('should show tutorial by default', () => {
+      const wrapper = createComponent();
+      const subHeader = wrapper.find('.help-panel__sub-panel-title');
+
+      assert.equal(subHeader.text(), 'Getting started');
+      assert.isTrue(wrapper.find('Tutorial').exists());
+      assert.isFalse(wrapper.find('VersionInfo').exists());
+    });
+
+    it('should show navigation link to versionInfo sub-panel', () => {
+      const wrapper = createComponent();
+      const link = wrapper.find('.help-panel__sub-panel-link');
+
+      assert.equal(link.text(), 'About this version');
+    });
+
+    it('should switch to versionInfo sub-panel when footer link clicked', () => {
+      const wrapper = createComponent();
+      wrapper.find('.help-panel__sub-panel-link').simulate('click');
+
+      assert.equal(
+        wrapper.find('.help-panel__sub-panel-title').text(),
+        'About this version'
+      );
+      assert.isTrue(wrapper.find('VersionInfo').exists());
+      assert.equal(
+        wrapper.find('VersionInfo').prop('versionData'),
+        fakeVersionDataObject
+      );
+      assert.isFalse(wrapper.find('Tutorial').exists());
+    });
+  });
+
+  context('when viewing versionInfo sub-panel', () => {
+    it('should show navigation link back to tutorial sub-panel', () => {
+      const wrapper = createComponent();
+      wrapper.find('.help-panel__sub-panel-link').simulate('click');
+
+      const link = wrapper.find('.help-panel__sub-panel-link');
 
-    angular.module('h', []).component('helpPanel', require('../help-panel'));
+      assert.isTrue(wrapper.find('VersionInfo').exists());
+      assert.isFalse(wrapper.find('Tutorial').exists());
+      assert.equal(link.text(), 'Getting started');
+    });
+
+    it('should switch to tutorial sub-panel when link clicked', () => {
+      const wrapper = createComponent();
+
+      // Click to get to version-info sub-panel...
+      wrapper.find('.help-panel__sub-panel-link').simulate('click');
```

Comment:
I legit don't know that there's another way to get this component to change its internally-active sub-panel without interacting with it in this manner!

Reply:
I think thats the right way to do it